sshproxy.go

```diff
@@ -30,7 +31,7 @@ import (
 	"os/exec"
 	"path/filepath"
 
-	"github.com/kr/pty"
+	"github.com/resin-io-playground/gexpect/pty"
```

Comment:
Why do we fork gexpect/pty ? If it's a proper fork we should move it under resin-io-modules